nestkernel/connector_model.h

```diff
@@ -115,57 +135,59 @@ class ConnectorModel
   bool
   is_primary() const
   {
-    return is_primary_;
+    return has_property( properties_, ConnectionModelProperties::IS_PRIMARY );
   }
 
   bool
   has_delay() const
   {
-    return has_delay_;
+    return has_property( properties_, ConnectionModelProperties::HAS_DELAY );
   }
 
   bool
   requires_symmetric() const
   {
-    return requires_symmetric_;
+    return has_property( properties_, ConnectionModelProperties::REQUIRES_SYMMETRIC );
   }
 
   bool
   requires_clopath_archiving() const
   {
-    return requires_clopath_archiving_;
+    return has_property( properties_, ConnectionModelProperties::REQUIRES_CLOPATH_ARCHIVING );
   }
 
   bool
   requires_urbanczik_archiving() const
   {
-    return requires_urbanczik_archiving_;
+    return has_property( properties_, ConnectionModelProperties::REQUIRES_URBANCZIK_ARCHIVING );
   }
 
   bool
   supports_wfr() const
   {
-    return supports_wfr_;
+    return has_property( properties_, ConnectionModelProperties::SUPPORTS_WFR );
   }
 
+  bool
+  has_flag_set( const ConnectionModelProperties& flag )
+  {
+    return has_property( properties_, flag );
+  }
```

Comment:
Rename and make const
```suggestion
  bool
  has_property( const ConnectionModelProperties& property ) const
  {
    return has_property( properties_, property );
  }
```

Reply:
:+1: